extensions/amp-analytics/0.1/vendors.js

```diff
@@ -40,6 +40,11 @@ export const ANALYTICS_CONFIG = {
       'scrollHeight': 'SCROLL_HEIGHT',
       'screenWidth': 'SCREEN_WIDTH',
       'screenHeight': 'SCREEN_HEIGHT'
+      // TODO: Requested by Parse.ly
+      //   screenAvailWidth (screen.availWidth)
+      //   screenAvailHeight (screen.availHeight)
+      //   colorDepth (screen.colorDepth)
```

Comment:
The above variables captured by this issue: https://github.com/ampproject/amphtml/issues/1503